index.json

```diff
@@ -10,5 +10,31 @@
     "load_before": [],
     "load_after": [],
     "authors": ["Dogeek <simon.bordeyne@gmail.com>"]
+  },
+  {
+    "url": "deselect-all.tar.gz",
+    "id": "deselect-all",
+    "name": "Deselect All",
+    "description": [
+      "Adds a button to deselect all highlighted cards."
+    ],
+    "version": "1.0.0",
+    "load_before": [],
+    "load_after": [],
+    "authors": ["Zei33 <balatro@zei.gg>"]
+  },
+  {
+    "url": "balatro-safety.tar.gz",
+    "id": "balatro-safety",
+    "name": "Balatro Safety",
+    "description": [
+		"Protects you from accidentally making",
```

Comment:
avoid having so much whitespace

Reply:
This is based on tab length. I will change my editor to use spaces instead of tabs.